foundation/foundation-upgrade/src/test/resources/test-files/v1_8_0/ExtensionsSecurityProjectsMigration/validation/IngestValidation.java

```diff
@@ -0,0 +1,81 @@
+package org.boozallen.aissemble;
+
+/*-
+ * #%L
+ * authzforce1::Pipelines::Spark Pipeline
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+
+import javax.enterprise.context.ApplicationScoped;
+
+import com.boozallen.aiops.security.authorization.AissembleKeyStore;
+import com.boozallen.aiops.security.authorization.AissembleKeycloakSecureTokenServiceClient;
+import com.boozallen.aiops.security.authorization.AissembleSimpleSecureTokenServiceClient;
+import com.boozallen.aiops.security.authorization.policy.AissembleAttribute;
+import com.boozallen.aiops.security.authorization.policy.AissembleAttributeProvider;
+import com.boozallen.aiops.security.exception.AissembleSecurityException;
```

Comment:
Are we not handling the imports are part of this ticket/migration? Are these imports still valid with the new class names?

Reply:
There is another migration that is already written in #194, this ticket is focusing on the authzforce and PDP objects. What I will do is rename the package names in the test file to emulate the other migration had already ran.